spec/dummy/config/routes.rb

```diff
@@ -0,0 +1,3 @@
+Rails.application.routes.draw do
```

Comment:
Just noting this comment in reference to needing routes or not ( for the dummy app as well. ) https://github.com/ManageIQ/manageiq-api-common/pull/4/files#r243638109